beetsplug/fetchart.py

```diff
@@ -454,11 +467,11 @@ def get(self, album, plugin, paths):
 
         matches = []
         # can there be more than one releasegroupid per response?
-        for mbid, art in data.get(u'albums', dict()).items():
+        for mbid, art_ in data.get(u'albums', dict()).items():
```

Comment:
I think it's fine for `art` to shadow the module name here?

Reply:
Reverted, and added ``# noqa F402`` to the line.